### ISSUE TITLE
Error `attempt to call method '_get_hl_from_capture' (a nil value)` in Nightly

### ISSUE THREAD

**Issue by datwaft** [NarutoXY/dim.lua#18]
The following error is displayed in the latest version of Neovim Nightly:

```error
Error detected while processing DiagnosticChanged Autocommands for "*":
E5108: Error executing lua ...al/share/nvim/site/pack/packer/start/dim.lua/lua/dim.lua:121: attempt to call method '_get_hl_from_capture' (a nil value)
stack traceback:
        ...al/share/nvim/site/pack/packer/start/dim.lua/lua/dim.lua:121: in function 'fn'
        ...1/share/nvim/runtime/lua/vim/treesitter/languagetree.lua:210: in function 'for_each_tree'
        ...al/share/nvim/site/pack/packer/start/dim.lua/lua/dim.lua:95: in function 'get_treesitter_hl'
        ...al/share/nvim/site/pack/packer/start/dim.lua/lua/dim.lua:47: in function 'highlight_word'
        ...al/share/nvim/site/pack/packer/start/dim.lua/lua/dim.lua:140: in function 'hig_unused'
        [string ":lua"]:1: in main chunk
        [C]: in function 'nvim_exec_autocmds'
        ...HEAD-9a5ac06_1/share/nvim/runtime/lua/vim/diagnostic.lua:727: in function 'set'
        ...-9a5ac06_1/share/nvim/runtime/lua/vim/lsp/diagnostic.lua:206: in function 'handler'
        ...neovim/HEAD-9a5ac06_1/share/nvim/runtime/lua/vim/lsp.lua:1056: in function ''
        vim/_editor.lua: in function <vim/_editor.lua:0>
```

This could be related to the breaking change https://github.com/neovim/neovim/issues/14090#issuecomment-1257184803 as I can see in the code that that function is used:

https://github.com/NarutoXY/dim.lua/blob/1fef581aaa3afebe0cfa8cb93ff747b107cb86be/lua/dim.lua#L80-L86

**Comment by amarakon** [comment]
I am having the same issue.

**Comment by bmulholland** [comment]
Not just in nightly, now also released in neovim 0.8
